NEWS.md

```diff
@@ -47,7 +44,7 @@ or that the queries dataset contains duplicates. (#2543)
 
 - `derive_vars_atc()` and `create_single_dose_dataset()` `by_vars` argument updated to use `get_admiral_option("subject_keys")` instead of  `USUBJID` or `STUDYID` in `bds_exposure.Rmd`. (#2501)
   
-- test scripts, R, and markdown files for `create_single_dose_dataset` and `occds.Rmd` updated to include a `STUDYID` column because of `get_admiral_option("subject_keys")` update above. (#2501)
+- The test scripts, R, and markdown files for `create_single_dose_dataset()` and `occds.Rmd` updated to include a `STUDYID` column because of `get_admiral_option("subject_keys")` update above. (#2501)
 
 - Update `derive_vars_period()` to make it work when there is only one new variable. (#2582)
 
```

Comment:
Should the next item be moved to the deprecation item?

Reply:
ummm...which item?  line 45?

Reply:
Line 51, removed arguments from `get_summary_records()`.

Reply:
yes good idea!